doc/api_assets/style.css

```diff
@@ -1,60 +1,49 @@
 /*--------------------- Layout and Typography ----------------------------*/
 html {
+  font-size: 1rem;
+  overflow-wrap: break-word;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
   -webkit-font-variant-ligatures: none;
           font-variant-ligatures: none;
 }
 
+* {
+  box-sizing: border-box;
+}
+
 body {
   font-family: "Lato", "Lucida Grande", "Lucida Sans Unicode", "Lucida Sans", Verdana, Tahoma, sans-serif;
-  font-size: 62.5%;
   margin: 0;
   padding: 0;
   color: #333;
   background: #fff;
 }
 
 h1, h2, h3, h4 {
-  margin: .8em 0 .5em;
+  margin: .8rem 0 .5rem;
   line-height: 1.2;
 }
 
 h5, h6 {
-  margin: 1em 0 .8em;
+  margin: 1rem 0 .8rem;
   line-height: 1.2;
 }
 
-h1 {
-  margin-top: 0;
-  font-size: 2.441em;
-}
-
-h2 {font-size: 1.953em;}
-
-h3 {font-size: 1.563em;}
-
-h4 {font-size: 1.25em;}
-
-h5 {font-size: 1em;}
-
-h6 {font-size: .8em;}
-
 pre, tt, code, .pre, span.type, a.type {
   font-family: Monaco, Consolas, "Lucida Console", monospace;
+  font-size: .9em;
```

Comment:
No `rem` here?

Reply:
This is the one place where `em` is needed so it does not dramatically downsize `code` tags inside headings (which can be up to `2rem`, and a `.9rem` code tag would look very bad in them).